plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -880,7 +880,7 @@ bool chain_plugin::import_reversible_blocks( const fc::path& reversible_dir,
    reversible_blocks.open( reversible_blocks_file.generic_string().c_str(), std::ios::in | std::ios::binary );
 
    reversible_blocks.seekg( 0, std::ios::end );
-   uint64_t end_pos = reversible_blocks.tellg();
+   int64_t end_pos = reversible_blocks.tellg();
    reversible_blocks.seekg( 0 );
```

Comment:
`tellg()` returns a `basic_istream<>::pos_type` I think I would rather see `auto` in this case.